bin/ci-bootstrap

```diff
@@ -9,9 +9,11 @@ app="${1:-hello}"
 # Run the bootstrap.yml playbook with the ansible vault password
 # passed to stdin
 echo "arnold" | docker run --rm -i \
+  -u "$(id -u)" \
```

Comment:
I don't know if it's the best hing to do. If we want to share `~/.kube` in the container we have to make it readable. We can change the right or change the user in the container. I choose the second one, WDYT ?